src/test/java/com/commercetools/sync/taxcategories/utils/TaxCategorySyncUtilsTest.java

```diff
@@ -0,0 +1,46 @@
+package com.commercetools.sync.taxcategories.utils;
+
+import com.commercetools.sync.taxcategories.TaxCategorySyncOptions;
+import io.sphere.sdk.commands.UpdateAction;
+import io.sphere.sdk.taxcategories.TaxCategory;
+import io.sphere.sdk.taxcategories.TaxCategoryDraft;
+import io.sphere.sdk.taxcategories.TaxCategoryDraftBuilder;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static com.commercetools.sync.taxcategories.utils.TaxCategorySyncUtils.buildActions;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+class TaxCategorySyncUtilsTest {
+
+    private TaxCategory old;
```

Comment:
what do you mean by `old`, maybe renaming these mock models with their actual name would be better, like `taxCategory`, `taxCategoryDraft`.